packages/core/src/CardanoNode/util/bufferChainSyncEvent.ts

```diff
@@ -0,0 +1,50 @@
+import { Observable } from 'rxjs';
+import { RequestNext } from '../types';
+
+export interface WithRequestNext {
```

Comment:
This already exists

Reply:
fd7ef848c and 7328a70cc should solve this